tools/ci_build/build.py

```diff
@@ -261,7 +261,6 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                  "-Donnxruntime_DEV_MODE=ON",
                  "-DPYTHON_EXECUTABLE=" + sys.executable,
                  "-Donnxruntime_USE_CUDA=" + ("ON" if args.use_cuda else "OFF"),
-                 "-Donnxruntime_CUDA_HOME=" + (cuda_home if args.use_cuda else ""),
```

Comment:
It looks like cuda_home is only used when (not is_windows) is true? Do wee need to remove it from the CI yaml file? or it is still needed for CMake?

Reply:
Keep it as an arg of build.py. But this line should be deleted. Thanks.